source/cities/CMakeLists.txt

```diff
@@ -4,12 +4,11 @@ SET(BOOST_LIBS ${Boost_PROGRAM_OPTIONS_LIBRARY} ${Boost_FILESYSTEM_LIBRARY}
 add_executable(cities cities.cpp)
 target_link_libraries(cities transportation_data_import connectors types ${PQXX_LIB} ${OSMPBF} pb_lib utils
     ${BOOST_LIBS} log4cplus z protobuf)
-INSTALL_TARGETS(/usr/bin/ cities)
-
-INSTALL_FILES(/usr/share/navitia/cities/alembic FILES alembic.ini alembic/env.py)
+install(TARGETS cities DESTINATION ${CMAKE_INSTALL_PREFIX})
```

Comment:
Should be `$INSTALL_PREFIX/bin`

Reply:
`$CMAKE_INSTALL_PREFIX` should not be used as it's automatically added by cmake when the destination is a relative path. 
I would rather have `install(TARGETS cities DESTINATION bin)` to have more flexibility when installing with `DESTDIR` like `make DESTDIR=/tmp install`